timeseries/src/main/java/com/hurence/logisland/timeseries/sax/SaxAnalyzer.java

```diff
@@ -0,0 +1,75 @@
+package com.hurence.logisland.timeseries.sax;
+
+import com.sun.javafx.collections.MappingChange;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.*;
+
+
+public class SaxAnalyzer {
+
+    public static Map<Integer, Integer> countOccurrence(int[] numbersToProcess) {
+        int[] possibleNumbers = new int[30];
```

Comment:
why 30 ?

Reply:
26 the number of alphabetic characters. the longest distance between characters would be 26. to make shure that we don't get an error I puted 30.